src/fakeredis_cluster.erl

```diff
@@ -146,23 +146,21 @@ handle_cast(_, State) ->
 handle_call(cluster_slots, _From, State) ->
     ?LOG("Handling a CLUSTER SLOTS request"),
     {reply, {ok, create_cluster_slots_resp(State)}, State};
-handle_call({start_instance, Port}, _From, #state{max_clients = MaxClients} = State) ->
-    ?LOG("start_instance: ~p", [Port]),
-    fakeredis_instance_sup:start_acceptors(Port, MaxClients),
+handle_call({restart_node, Port}, _From,
+            #state{max_clients = MaxClients, options = Options} = State) ->
+    fakeredis_instance_sup:start_link(Port, Options, MaxClients),
     {reply, ok, State};
-handle_call({kill_instance, Port}, _From, State) ->
-    ?LOG("kill_instance: ~p", [Port]),
-    Pids = gproc:lookup_pids({p, l, {local, Port}}),
-    ?LOG("Stop port=~p, pids=~p", [Port, Pids]),
-    [exit(Pid, kill) || Pid <- Pids],
+handle_call({stop_node, Port}, _From, State) ->
+    Pids = gproc:lookup_pids({n, l, {fakeredis_instance_sup, Port}}),
+    [exit(Pid, normal) || Pid <- Pids],
     {reply, ok, State};
 handle_call({get_redirect_by_key, Key}, _From, State) ->
     Slot = fakeredis_hash:hash(Key),
     case State#state.ask_redirects of
         #{Key := Id} ->
             %% ASK redirect exists for this key
             #node{address = Addr,
-                  port    = Port} = maps:get(Id, State#state.node_map),
+                  port    = Port} = maps:get(Id, State#state.nodes),
```

Comment:
"Nodes" somehow suggests that it's a list, but it's a map where the keys are IDs. Maybe "node_by_id" or something like that would be a better naming convention for maps. Just a general thought. I don't think there is any confusion here. Keep "nodes".